### PR TITLE
[Release] Update `paging` Anyscale API call

### PR DESCRIPTION
**Pull request by Yard1**
## Why are these changes needed?

Fixes release test infra being broken due to a change in Anyscale API, updates Anyscale package.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/ray_release/util.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def reinstall_anyscale_dependencies() -> None:
 
     # Copy anyscale pin to requirements.txt and requirements_buildkite.txt
     subprocess.check_output(
-        "pip install -U anyscale==0.5.76",
+        "pip install -U anyscale==0.5.81",
```
Comment (AmeerHajAli) [inline release/ray_release/util.py:147]: do we want to fix the anyscale version? wouldn't it be better to always install the latest anyscale version?

File: release/requirements.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Requirements to run release tests from the local machine (including client tests)
 click
 # Copy anyscale pin to requirements_buildkite.txt and util.py
-anyscale==0.5.76
+anyscale==0.5.81
```
Comment (AmeerHajAli) [inline release/requirements.txt:4]: dditto

File: release/requirements_buildkite.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Requirements to run release tests from buildkite (client dependencies will be installed separately)
 # Copy anyscale pin to requirements.txt and util.py
-anyscale==0.5.76
+anyscale==0.5.81
```
Comment (AmeerHajAli) [inline release/requirements_buildkite.txt:3]: ditto

**Review by AmeerHajAli — APPROVED** [review]

File: release/ray_release/util.py (hunk 1 above)
Comment (Yard1) [reply release/ray_release/util.py:147]: We have always fixed the anyscale version - let's not change that in this PR, but definitely something we should consider overall - I'll discuss with the rest of the oss infra team

**Pull request closed without merging**